app/views/partials/_header.html.erb

```diff
@@ -15,7 +15,9 @@
         <li>
           <%= link_to t('.forms'), services_path %>
         </li>
-
+        <li>
+        //  <%= link_to t('.support'), t('.supportURL') %>
+        </li>
```

Comment:
Not quite sure what's intended here.
Are the // characters intended to comment out this line?

If so, worth noting:

In Javascript...
// this is a single line comment
or 
/*
  this is a multiple
  line comment
  */
  
  In HTML...
  <!-- This is a comment -->
  
  In Ruby .erb files...
<%# this will comment out link_to t('blah') %>

Reply:
sorry was meant as a marker to see where I was adding stuff… wasn't meant to stay there!